api/route/Article/article.recordImageGen.ts

```diff
@@ -0,0 +1,30 @@
+import { Route } from '..';
+import { Error, Success } from '../router';
+import { Article, Category } from '@shared';
+import { string } from 'pg-format';
+import { getClient } from '../../database';
+
+export const recordImageGen: Route = (req, res) => {
+    const { id } = req.params;
+
+  const recordImage = async (article_id: any) => {
+    try {
+      await getClient().query(
+        `UPDATE article
+         SET image_gen=true
+         WHERE id=$1`,[article_id]
```

Comment:
Parameterized queries! @AardhynLavender would be proud of you
<sub>noice</sub>

Reply:
I think any developer would be pleased to see industry practice in code. Don't you think, @ArtemKech

Reply:
Ahahah, I know. But people keep forgetting or missing this thing :)